Confus/Confus/Game.cpp

```diff
@@ -75,95 +73,106 @@ namespace Confus
         m_LevelRootNode->setScale(irr::core::vector3df(1.0f, 1.0f, 1.0f));
         m_LevelRootNode->setVisible(true);
         m_Device->setEventReceiver(m_EventManager);
-		updateSceneTransformations();
+        updateSceneTransformations();
         initializeLevelColliders();
 
         m_PlayerNode.setEventManager(m_EventManager);
         m_Device->getCursorControl()->setVisible(false);
 
-        m_Connection->addFunctionToMap(static_cast<RakNet::MessageID>(Networking::EPacketType::MainPlayerJoined), [this](RakNet::Packet* a_Data)
+        m_Connection->addFunctionToMap(static_cast<RakNet::MessageID>(Networking::EPacketType::MainPlayerJoined), [this](RakNet::BitStream* a_Data)
```

Comment:
I think that the functionmap still works with packets, instead of bitstreams. (on develop at least)